migrator/migrations/n_52_to_n_53_postgres_simple_access_scopes/migration.go

```diff
@@ -251,6 +255,52 @@ func migrateRoles(gormDB *gorm.DB, postgresDB *pgxpool.Pool, legacyStore legacyr
 	return nil
 }
 
+func getReportConfigurationScopeID(reportConfiguration *storage.ReportConfiguration) (string, error) {
+	reportConfigurationScopeID := strings.TrimPrefix(reportConfiguration.GetScopeId(), accessScopeIDPrefix)
+	if replacement, found := accessScopeIDMapping[reportConfigurationScopeID]; found {
+		reportConfigurationScopeID = replacement
+	}
+	_, accessIDParseErr := uuid.FromString(reportConfigurationScopeID)
+	if accessIDParseErr != nil {
+		log.WriteToStderrf("failed to convert report configuration to postgres format, bad scope ID. Report Configuration [%s], error %v", reportConfiguration.GetName(), accessIDParseErr)
+		return "", accessIDParseErr
+	}
+	return reportConfigurationScopeID, nil
+}
+
+func updateReportConfigurationScopeIDs(postgresDB *pgxpool.Pool) error {
```

Comment:
nit: would be nice to have a comment specifying that this is has been added at a later point (although, I don't think it's _that_ critical to have).